src/gui/systray.h

```diff
@@ -61,6 +61,9 @@ class Systray
     Q_INVOKABLE void setOpened();
     Q_INVOKABLE void setClosed();
     Q_INVOKABLE void positionWindow(QQuickWindow *window) const;
+    Q_INVOKABLE bool hasNotification() const { return !_notificationList.isEmpty(); }
+    Q_INVOKABLE QString getLastNotification() const;
```

Comment:
Please make this a Q_PROPERTY. This allows to write the QML code in a declarative instead of imperative manner